pages/configuration/logs.mdx

```diff
@@ -71,6 +71,31 @@ If you installed Memgraph using Docker:
     cat /var/log/memgraph/memgraph_2022-03-04.log
     ```
 
+    #### Filtering unnecessary log messages
+
+    Memgrpah uses several log files:
+    - General Logs: These are the default logs that track various levels such as `TRACE`, `DEBUG`, `INFO`, `WARNING`, `ERROR`, `CRITICAL`. 
+    - Audit Logs: These logs contain [records of all queries](/configuration/audit-log) executed on the database when enabled.
+    - Write-ahead Logs (WAL): These logs record [each database modification](/configuration/data-durability-and-backup) before it's written to the database, helping to reconstruct the database's most recent state.

```

Comment:
Is this part necessary? With my comment on the issue, I meant it would be good to explain how with the `TRACE` level logs you get all logs, also the ones that Lab runs. To exclude those, use the below command. (just as the issue states)

Also, this whole page refers to the general logs, so I feel like it's overkill to mention others here.